test/mocks/weets.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-unused-vars */
+
+const { BAD_REQUEST } = require('../../app/errors');
+const { WEET_CREATED } = require('../../config/constants');
+
+const createWeetInput = {
+  content: 'No statement can catch the ChuckNorrisException'
+};
+
+const createBadWeedInput = {
+  content:
+    'abcdefghi abcdefghi abcdefghi abcdefghi abcdefghi abcdefghi abcdefghi abcdefghi abcdefghi abcdefghi abcdefghi abcdefghi abcdefghi abcdefghi a'
```

Comment:
To prevent this non-clean code, you could use a library to generate random content like [faker.js](https://github.com/marak/Faker.js/). It's up to you

Reply:
Thank you bro, great tool.